### PR TITLE
Add jquery.turbolinks v2.1.0

### LINKED ISSUE #7254: Update jquery.turbolinks
https://cdnjs.com/libraries/jquery.turbolinks
https://github.com/kossnocorp/jquery.turbolinks

### PR DESCRIPTION
**Pull request by x09326**
PR for #7254
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.
# Profile of the lib
- Git repository (required): https://github.com/kossnocorp/jquery.turbolinks
- Official website (optional, not the repository):
- NPM package url (optional): https://www.npmjs.com/package/jquery.turbolinks
- GitHub / Bitbucket popularity (required):
  - Count of stars: 921
  - Count of watchers: 25
  - Count of forks: 50
- NPM download stats (optional):
  - Downloads in the last day: 0
  - Downloads in the last week: 154
  - Downloads in the last month: 385
# Essential checklist
- [ ] I'm the author of this library
  - [ ] I would like to add link to the page of this library on CDNJS on website / readme
- [ ] This lib was not found on cdnjs repo
- [x] No already exist / duplicated issue and PR
- [x] The lib has notable popularity
  - [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  - [ ] More than 500 downloads stats per month on npm registry
- [x] Project has public repository on famous online hosting platform (or been hosted on npm)
# Auto-update checklist
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.
# Git commit checklist
- [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
- [x] The parent of the commit(s) in the PR is not old than 3 days.
- [x] Pull request is sending from a non-master branch with meaningful name.
- [x] Separate unrelated changes into different commits.
- [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
- [x] Close corresponding issue in commit message
- [x] Mention related issue(s), people in commit message, comment.

with npm auto-update config added, close #7254

### CODE REVIEW

**Comment by x09326** [comment]
@dannyxx001 Please help me review this PR, thank you.
I couldn't find the ```homepage```, so I didn't add it.

File: ajax/libs/jquery.turbolinks/package.json [hunk 1]
```diff
@@ -1,8 +1,29 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kossnocorp/jquery.turbolinks.git"
+  },
+  "npmName": "jquery.turbolinks",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "vendor/assets/javascripts/*"
```
Comment (dannyxx001) [inline ajax/libs/jquery.turbolinks/package.json]: I think you can use "vendor/assets/javascripts/" as basepath and "jquery.turbolinks*" as files in npmFileMap, but I'm not very sure. @PeterDaveHello Would you please explain again that whether only "dist" or "build" can be basepath? Thank you.

Comment (PeterDaveHello) [inline ajax/libs/jquery.turbolinks/package.json]: In this case, `vendor/assets/javascripts/` or `vendor/assets/` could be the base path, maybe you can discuss with its author and see if it's possible add a `dist` folder for release files?

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Pull request closed without merging**